scripts/ingests/ultracool_sheet/Ingest_radial_velocities.py

```diff
@@ -0,0 +1,112 @@
+from astrodb_utils import load_astrodb, find_source_in_db, AstroDBError, ingest_publication
+import logging
+from astropy.io import ascii
+from urllib.parse import quote
+import requests
+from astropy.table import Table
+import sys
+sys.path.append( '.' )
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+from math import isnan
+import sqlalchemy.exc
+
+
+
+logger = logging.getLogger("AstroDB")
+logger.setLevel(logging.INFO)
+
+
+RECREATE_DB = True
+db = load_astrodb("SIMPLE.sqlite", recreatedb=RECREATE_DB, reference_tables=REFERENCE_TABLES)
+
+# Load Ultracool sheet
+doc_id = "1i98ft8g5mzPp2DNno0kcz4B9nzMxdpyz5UquAVhz-U8"
+sheet_id = "361525788"
+link = f"https://docs.google.com/spreadsheets/d/{doc_id}/export?format=csv&gid={sheet_id}"
+
+# read the csv data into an astropy table
+uc_sheet_table = ascii.read(
+    link,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False,
+    delimiter=",",
+)
+
+# Load Ultracool sheet refrences
+sheet_id = "453417780"
+link = f"https://docs.google.com/spreadsheets/d/{doc_id}/export?format=csv&gid={sheet_id}"
+
+# read the csv data into an astropy table
+uc_reference_table = ascii.read(
+    link,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False,
+    delimiter=",",
+)
+
+uc_ref_to_ADS = {}
+for ref in uc_reference_table:
+    uc_ref_to_ADS[ref["code_ref"]] = ref["ADSkey_ref"]
+
+def uc_ref_to_simple_ref(ref):
+    t=(
+        db.query(db.Publications)
+        .filter(db.Publications.c.bibcode==uc_ref_to_ADS[ref])
+        .astropy()
+    )
+    if(len(t)==0):
+        ingest_publication(db,bibcode=uc_ref_to_ADS[ref],reference=ref)
+        return ref
+    else:
+        return t["reference"][0]
```

Comment:
At first, I didn't see where this function was being called. Maybe a comment/docstring about what this function does and where it will be used would be useful.